apps/src/netsim/NetSimTable.js

```diff
@@ -18,6 +18,7 @@
 var _ = require('../utils').getLodash();
 var ObservableEvent = require('../ObservableEvent');
 var NetSimApi = require('./NetSimApi');
+var netsimGlobals = require('./netsimGlobals');
```

Comment:
I think this should be capitalized for consistency with the other "modules" you are are requiring.